.github/workflows/deploy-docs.yml

```diff
@@ -0,0 +1,72 @@
+# Preamble
+name: Deploy SaltProc docs
+
+on:
+  push:
+    branches:
+      - master 
+    paths:
+      - 'doc/**'
+      - 'saltproc/**'
+
+env:
+  CACHE_NUMBER: 0 #change to manually reset cache
+
+jobs:
+  deploy-docs:
+    runs-on: ubuntu-latest
+      #if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    defaults:
+      run:
+        shell: bash -l {0}
+
+    steps:
+      - uses: actions/checkout@v2
+
+# Setup conda 
+      - name: Set up conda
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          miniforge-variant: Mambaforge
+          miniforge-version: latest
+          activate-environment: saltproc-env
+          use-mamba: true
```

Comment:
same here about using mamba (re: dependency resolution). If there's an issue, we should add an inline comment about this choice.